d3b_utils/database.py

```diff
@@ -0,0 +1,17 @@
+import psycopg2
+
+
+def test_pg_connection(db_url):
+    """Check if it is possible to connect to a postgres database
+
+    :param db_url: database url to attempt to connect to
+    :type db_url: str
+    :return: outcome of if it is possible to connect to the specified database
+    :rtype: bool
+    """
+    try:
+        conn = psycopg2.connect(db_url, connect_timeout=1)
+        conn.close()
+        return True
+    except psycopg2.OperationalError:
```

Comment:
It might be useful to show the user the error encountered instead of just returning `False`

Reply:
I guess if that's the case, i may as well return _any_ error: 
```suggestion
    except Error as e:
        return {"connected":False, "error": e}
```

Reply:
>    except Error as e:

`Exception`?